client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -179,6 +194,31 @@ const convertToApiPackage = pckg => {
 		'weight',
 		'signature',
 	] );
+	if ( customsItems ) {
+		apiPckg.contents_type =
+			'other' !== pckg.contentsType ? pckg.contentsType || 'merchandise' : pckg.contentsExplanation;
+		apiPckg.restriction_type =
+			'other' !== pckg.restrictionType
+				? pckg.restrictionType || 'none'
+				: pckg.restrictionExplanation;
+		apiPckg.abandon_on_non_delivery = Boolean( pckg.abandonOnNonDelivery );
+		apiPckg.itn = pckg.itn || '';
+		const getProductValue = productId =>
+			isNil( customsItems[ productId ].value )
+				? getProductValueFromOrder( state, productId, orderId, siteId )
+				: customsItems[ productId ].value;
+
+		apiPckg.items = uniqBy( pckg.items, 'product_id' ).map( ( { product_id } ) => ( {
+			description: customsItems[ product_id ].description,
+			quantity: sumBy( filter( pckg.items, { product_id } ), 'quantity' ),
+			value: getProductValue( product_id ),
```

Comment:
This should be the total value of the given item in the package ([EasyPost docs](https://www.easypost.com/customs-guide.html)):
> value = Total value in US dollars of all the items of that type in the package

Should this line be something like `getProductValue( product_id ) * {product_quantity_in_package}`?

Similar issue with the weight below, `weight: customsItems[ product_id ].weight`

Unit tests will help here.

Reply:
PR opened: https://github.com/Automattic/wp-calypso/pull/26289